### PR TITLE
Add ppc64le arch to weave net

### PR DESCRIPTION
**Pull request by mkumatag**
Now `ppc64le` arch is supported from `weaveworks/weave:2.1.0` version, so doc needs to be updated with architecture support, hence submitting this PR.

---
This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/website/6814)

### CODE REVIEW

**Comment by mkumatag** [comment]
@tengqm any idea why travis job is failing? any other action required on my end?

File: docs/setup/independent/create-cluster-kubeadm.md [hunk 1]
```diff
@@ -332,7 +332,7 @@ please see [here](https://kubernetes.io/docs/concepts/cluster-administration/net
 
 The official Weave Net set-up guide is [here](https://www.weave.works/docs/net/latest/kube-addon/).
 
-**Note:** Weave Net works on `amd64`, `arm` and `arm64` without any extra action required.
+**Note:** Weave Net works on `amd64`, `arm` , `arm64` and `ppc64le` without any extra action required.
```
Comment (zhangxiaoyu-zidif) [inline docs/setup/independent/create-cluster-kubeadm.md]: no space between "arm" and ","

**Comment by tengqm** [comment]
@mkumatag please try a small fix and push again. Looks like the gate was experiencing some transient problems.

File: docs/setup/independent/create-cluster-kubeadm.md (hunk 1 above)
Comment (mkumatag) [reply docs/setup/independent/create-cluster-kubeadm.md]: fixed it.!

**Pull request merged**